PowerFGT/Public/Connection.ps1

```diff
@@ -74,7 +74,9 @@ function Connect-FGT {
         [Parameter(Mandatory = $false)]
         [string[]]$vdom,
         [Parameter(Mandatory = $false)]
-        [boolean]$DefaultConnection=$true
+        [boolean]$DefaultConnection = $true,
+        [Parameter(Mandatory = $false)]
+        [int]$Timeout = 5
```

Comment:
can you don't set default value ?

Reply:
I set the default to 0. That is the default of Invoke-Webrequest. Not sure on adding a whole if-then check just for this parameter. Is there another way you think it could be accomplished?